core/aws-core/src/main/java/software/amazon/awssdk/awscore/client/builder/AwsDefaultClientBuilder.java

```diff
@@ -422,6 +424,20 @@ private void configureRetryPolicy(SdkClientConfiguration.Builder config) {
     private void configureRetryStrategy(SdkClientConfiguration.Builder config) {
```

Comment:
What about DynamoDB or other service-specific retry strategy configurations? How do we include those as well as defaults, if the customer hasn't overridden the defaults?

Reply:
For DynamoDB, or any other service with custom retry strategy we codegen the setting as in 

```
        builder.option(SdkClientOption.RETRY_STRATEGY, DynamoDbRetryPolicy.resolveRetryStrategy(config));
```

So they will have a non-empty set of retry conditions already set. And the specified method should take care of setting the correct defaults already. E.g., for DynamoDB the method already returns

```

        return AwsRetryStrategy.forRetryMode(retryMode)
            .toBuilder()
            .maxAttempts(MAX_ATTEMPTS)
            .backoffStrategy(exponentialDelay(BASE_DELAY, SdkDefaultRetrySetting.MAX_BACKOFF))
            .build();
```